db/seeds.rb

```diff
@@ -22,9 +25,29 @@ def seed_station_database(file_path)
   stations.each do |station|
     city = City.find_by(city: station[:city])
     station.delete_if { |k,v| k == :lat || k == :long || k == :city}
-    city.stations.create(station)
+    city.stations.create!(station)
   end
 end
 
-seed_city_database("db/csv/station.csv")
-seed_station_database("db/csv/station.csv")
+def seed_subscriptions_database(file_path)
+  subscriptions = from_csv(file_path)
+  subscriptions.each do |subscription_type|
+    subscription_type.keep_if {|k,v| k == :subscription_type}
+  end
+  subscriptions.uniq.each {|sub_type| Subscription.create!(sub_type)}
+end
+
+def seed_trips_database(file_path)
+  trips = from_csv(file_path)
+  trips.each do |trip|
+    subscription = Subscription.find_by(subscription_type: trip[:subscription_type])
+    trip.delete_if {|k,v| k == :subscription_type || k == :start_station_name || k == :end_station_name}
```

Comment:
I was confused about the same thing but if you take a look at your station database, it doesn't look like sql autogenerated a primary key. It uses the station_id in the csv as the primary key. Looking at the Trips csv, every start_station_id and end_station_id refers to the same info as the station_id. I added a pic with a few examples.
<img width="1440" alt="screen shot 2017-07-02 at 8 19 09 am" src="https://user-images.githubusercontent.com/25845955/27770645-a46aea5a-5eff-11e7-8106-b8a780ce9c50.png">